docs/release-notes/8.9.asciidoc

```diff
@@ -147,6 +147,7 @@ There are no breaking changes in 8.9.0.
 [discrete]
 [[features-8.9.0]]
 ==== New features
+* Introduces the `top` command for MacOS and Linux, which shows active processes that {elastic-endpoint} is monitoring and recording. Processes are sorted by how much CPU they are causing {elastic-endpoint} to consume. Learn more about the `top` command by referring to the (https://www.github.com/elastic/endpoint/blob/main/EndpointTopCommand.md[Readme]).
```

Comment:
Sorry, just one more suggested edit. Otherwise, everything looks good! 

```suggestion
* Introduces the `top` command for MacOS and Linux, which shows active processes that {elastic-endpoint} is monitoring and recording. Processes are sorted by how much CPU they are causing {elastic-endpoint} to consume. Learn more about the `top` command by referring to the (https://www.github.com/elastic/endpoint/blob/main/EndpointTopCommand.md[readme]).
```